### PR TITLE
Clarify email domains and subdomains on SSO config page

### PR DESCRIPTION
**Pull request by stephaurelio**
### Proposed changes

This PR addresses a question that came from a customer that needed clarity and an example of what the SSO docs mean when they tell users to include email domains and subdomains. In this PR, I added an example domain and subdomain. I also opted to style this as inline code to be consistent with domain examples in [this docs page](https://docs.docker.com/config/daemon/systemd/).

Page updated:
- `single-sign-on/configure/index.md`

### CODE REVIEW

File: single-sign-on/configure/index.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ Follow the steps on this page to configure SSO for your organization or company.
 
     >**Note**
     >
-    > Format your domains without protocol or www information, for example, yourcompany.com. This should include all email domains and subdomains users will use to access Docker. Public domains such as gmail.com, outlook.com, etc. aren’t permitted. Also, the email domain should be set as the primary email.
+    > Format your domains without protocol or www information, for example, `yourcompany.com`. This should include all email domains and subdomains users will use to access Docker, for example `yourcompany.com` and `us.yourcompany.com`. Public domains such as `gmail.com`, `outlook.com`, etc. aren’t permitted. Also, the email domain should be set as the primary email.
```
Comment (thaJeztah) [inline single-sign-on/configure/index.md:21]: It's best to use designated example domains for this (see https://www.rfc-editor.org/rfc/rfc2606.html#section-2), as domains like `yourcompany.com` can be actual domains owned by someone.

For example;

- `yourcompany.example.com`
- `company.example.com`
- `yourcompany.example`

**Review by thaJeztah — CHANGES_REQUESTED** [review]

File: single-sign-on/configure/index.md (hunk 1 above)
Comment (stephaurelio) [reply single-sign-on/configure/index.md:21]: Sure, I can make that update.

Comment (thaJeztah) [reply single-sign-on/configure/index.md]: Thanks!

I keep; forgetting to look at adding that to the docs guidelines 😅. In _most_ cases it's not problematic, but we've had some cases where we used an existing domain in the examples, and users (being users) copy/pasted the commands as-is (which isn't, ahem, "ideal" 😅

**Review by thaJeztah — APPROVED** [review]
domain changes LGTM, thanks!

File: single-sign-on/configure/index.md (hunk 1 above)
Comment (stephaurelio) [reply single-sign-on/configure/index.md]: That makes sense!

Comment (ChrisChinchilla) [reply single-sign-on/configure/index.md]: I can add this to our guide and style checkers at some point

Comment (aevesdocker) [reply single-sign-on/configure/index.md]: If you have the time @ChrisChinchilla, that'd be great

Comment (ChrisChinchilla) [reply single-sign-on/configure/index.md]: Yeah was just talking with @thaJeztah about it :)

**Pull request merged**